test/ComparedQueryable.Test/ComparedQueryableTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace ComparedQueryable.Test
+{
+    public class ComparedQueryableTest

```

Comment:
I didn't want to test this given that we have a separate test for the natural sort test algorithm. Otherwise, we end repeating the same test cases in both cases.